### DISCUSSION TITLE
Trame with Slicer 3D instead of Paraview

### DISCUSSION THREAD

**Discussion by kerim371** [Kitware/trame#633] [General] [answered]
Hi,

I know Paraview can be used as a backend with Trame wich is great.
But there is another project called [Slicer 3D](https://www.slicer.org/).

The question that arrives is were there any attemts to create something similar with Slicer 3D?
Maybe if I decide to make it by myself is it difficult to make it by analogy with Paraview or will it be completely different thing?

**Comment by jourdain** [comment]
@finetjul

**Comment by finetjul** [comment, marked as answer]
Yes, we are working towards making trame work with 3D Slicer. It's a lot of work though and it's not ready though. We will announce something early 2025. We are looking for funding if you are interested. Reach out to me in PM if you want more information.
